packages/environment/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": ["../../.eslintrc.package.json"],
+    "globals": {
+        "BUNDLE_CHAIN_FILENAME": "readonly"
```

Comment:
What's this for?

Reply:
ESlint complained about a no-undef ruld. It's to resolve [this global name](https://github.com/trufflesuite/truffle/blob/connect-or-start/packages/environment/develop.js#L13) introduced by our webpack config. The setting configures ESLint to recognize BUNDLE_CHAIN_FILENAME as valid variable name and check its usage is read only

Reply:
Oh, weird!  I had no idea about that.  Do you know, is this used anywhere else?  I'm wondering if it makes sense to put this in the global one.

Reply:
OK I just checked and it looks like it's not used anywhere else, so I guess it's fine to just put it here.

Reply:
Yea, I looked too. The ESLint config setup is very flexible now!